aws_guardduty_rules/aws_guardduty_high_sev_findings.py

```diff
@@ -1,10 +1,13 @@
+from panther_aws_helpers import aws_rule_context
+
 def rule(event):
     return 7.0 <= float(event.get("severity", 0)) <= 8.9
 
-
 def dedup(event):
     return event.get("id")
 
-
```

Comment:
[PEP8](https://peps.python.org/pep-0008/#blank-lines) disagrees with this -- there should be two lines surrounding top-level funcs and class definitions